packages/forklift-console-plugin/src/utils/index.ts

```diff
@@ -1,8 +1,10 @@
 // @index(['./*', /style/g], f => `export * from '${f.path}';`)
 export * from './constants';
 export * from './deepCopy';
+export * from './docLinks';
```

Comment:
we will remove those indexes soon, so guessing no need to add it as it just will create more work in the near future